Tasks/AzureRmWebAppDeployment/operations/AzureAppServiceUtility.ts

```
@@ -63,13 +63,21 @@ export class AzureAppServiceUtility {
     }
 
     public async pingApplication(): Promise<void> {
-        try {
-            var applicationUrl: string = await this.getApplicationURL();
+        var applicationUrl: string = await this.getApplicationURL();
```

I think you are changing the behavior by not ignoring the errors in getApplication

---

reverted back to the earlier behavior.